src/test/java/org/janelia/saalfeldlab/n5/AbstractN5Test.java

```diff
@@ -129,7 +130,10 @@ public void testCreateGroup() {
 
 		final Path groupPath = Paths.get(groupName);
 		for (int i = 0; i < groupPath.getNameCount(); ++i)
-			if (!n5.exists(groupPath.subpath(0, i + 1).toString()))
+			//replace backslashes in case of windows
```

Comment:
Shouldn't we rather override this in n5-hdf5?

Reply:
Windows accepts `/` for paths. There's a deeper question here if we should have `/` should be the canonical group separator for the N5 API and let `N5Writer.exists` convert the `/` to the operating system specific case.

I had a more ambitious rewrite of this test, but I thought a minimalist change that still uses the path API would suffice since this is just a test.

Reply:
I feel that disabling the ability to use `\` for Windows paths is a break in spec that requires a little more attention.  This is probably not very practically relevant but Windows supporting `/` as a path separator is a very old Java thing that predates how the `Filesystem` API pretends to handle this today.  My preferred solution to the failing test in n5-hdf5 is to override the test in n5-hdf5.